data/resources/gamedb.yaml

```diff
@@ -99027,9 +99027,9 @@ PAPX-90044:
   traits:
     - ForcePGXPCPUMode
 SCPS-45317:
-  name: "Metal Gear Solid (Asia) (Disc 1)"
+  name: "Metal Gear Solid (Japan, Asia) (Disc 1) (Ichi) (Premium Package)"
   discSet:
-    name: "Metal Gear Solid (Asia)"
+    name: "Metal Gear Solid (Japan, Asia) (Premium Package)"
```

Comment:
Hopefully this one wasn't actually being detected before, because otherwise it's going to change memory card names.

Reply:
Eh, don't worry about it. It's not like it deletes old cards, users would just have to rename them.